### PR TITLE
hwdb: add a click angle entry for the Logitech MX Anywhere 2s

### PR DESCRIPTION
**Pull request by whot**

### CODE REVIEW

File: hwdb/70-mouse.hwdb [hunk 1]
```diff
@@ -329,6 +329,10 @@ mouse:usb:v046dpc52b:name:Logitech Unifying Device. Wireless PID:1017:
 mouse:usb:v046dp1017:name:Logitech Anywhere MX:
  MOUSE_WHEEL_CLICK_ANGLE=20
 
+# Logitech Anywhere MX 2s
+mouse:usb:v046dp406a:name:Logitech MX Anywhere 2S:
+ MOUSE_WHEEL_CLICK_ANGLE=20
```
Comment (poettering) [inline hwdb/70-mouse.hwdb:332]: why not merge this entry with the entry three lines up?

**Comment by poettering** [comment]
I figure with this #10459 can be closed, right?

File: hwdb/70-mouse.hwdb (hunk 1 above)
Comment (whot) [reply hwdb/70-mouse.hwdb:332]: situational blindness... will update

**Comment by whot** [comment]
#10459 is the Anywhere 2, not the 2S and I still don't have confirmation over there that the hwdb entry they picked worked. So that one is an independent one.

**Comment by whot** [comment]
force-pushed with the update requested in https://github.com/systemd/systemd/pull/10555#pullrequestreview-169260836

**Pull request merged**